### ISSUE TITLE
Monero Community Workgroup Meeting: October 26th 15:00 UTC

### ISSUE THREAD

**Issue by plowsof** [monero-project/meta#1095]
Location: [Libera.chat, #monero-community](https://libera.chat/) | [Matrix](https://matrix.to/#/#monero-community:monero.social?via=matrix.org&via=monero.social)

[Instructions for joining the monero.social Matrix server.](https://www.getmonero.org/resources/user-guides/join-monero-matrix.html)

Time
15:00 UTC [Check your timezone](https://www.timeanddate.com/worldclock/converter.html)

Moderator: plowsof

Please reach out in advance of the meeting if you would like to propose an agenda item.

Proposed Meeting Items:

1. Introduction
2. Greetings
3. Community highlights    
    - escapethe3ra has received funding (independently!) for [monero.observer until 2026](https://monero.observer/monero-observer-funding-2025/)
        - [Dev activity report week 42](https://monero.observer/monero-dev-activity-report-week-42-2024/)
    - boog900 + SyntheticBird finding proxy rpc nodes  
    - @Monero twitter account requests community participation, what/who to tweet/retweet/follow
    - Goodell, Salazar, & Slaughter (2024). “Uniformly Most Powerful Tests for Ad Hoc Transactions in Monero.” has been posted: https://github.com/cypherstack/churn/releases/tag/final 
         - [Ruckniums review](https://github.com/cypherstack/churn/issues/2)    
    - News: [Monero Observer](https://www.monero.observer/) - [Revuo Monero](https://revuo-xmr.com/)
5. [CCS updates](https://ccs.getmonero.org/)    [My2cents](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/476#note_26817) on 'Rottenwheel comments on Monerobulls CCS reg progress / MB responds [beginning here](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/476#note_26752)'
  a. [Offline Signing Library for XmrSigner Production](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/495)    
  b. [Carrot animated video](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/506)    
  c. [Draft: Audit monero-serai and monero-wallet](https://gist.github.com/kayabaNerve/3723d0a3f2b62ef8ef00c0c4a574fb8e)
  d. [Suggestion for re purposing Outreach funds](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/70#note_26878)
  e. [mainnet_pat CCS proposal for completion of XMR-BCH atomic swaps](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/513)
6. Workgroup reports    
  a. Dev workgroup
  b. Localization workgroup
  c. Outreach workgroup
  d. Events workgroup - [MoneroKon 2025](https://monerokon.org)
  e. Website workgroup
  f. Policy workgroup
  g. Research workgroup
  h. [Seraphis Migration workgroup](https://github.com/seraphis-migration)
7. Open ideas time    
8. Confirm next meeting date/time    

[Previous meeting including logs](https://github.com/monero-project/meta/issues/1088)    

Meeting logs will be posted here afterwards.

**Comment by geonic1** [comment]
https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/70#note_26878

**Comment by plowsof** [comment]
Logs 
> __< plowsof >__ https://gist.github.com/kayabaNerve/3723d0a3f2b62ef8ef00c0c4a574fb8e is the current draft kayabanerve?      

> __< dEBRUYNE >__ syntheticbird: Can you PM me on the IRC side?     

> __< dEBRUYNE >__ shortwavesurfer2009: I am fairly certain Bit2Me uses Kraken for liquidity and that's why they chose to delist it     

> __< midipoet >__ dEBRUYNE: do you have a contact with anybody in Kraken?     

> __< plowsof >__ Re-sharing vthors xmrSigner related proposal as its changed alot https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/495     

> __< plowsof >__ There is a final milestone, ' stretch goals ', which attempts to account for/use any possible overfunding which ive not seen before and  also needs feedback from community and luigi1111     

> __< vthor >__ Thank you very much plowsof :)     

> __< k​ayabanerve:monero.social >__ Yes plowsof     

> __< plowsof >__ meeting in 2 hours https://github.com/monero-project/meta/issues/1095      

> __< k​ayabanerve:matrix.org >__ plowsof: Thanks for adding my item :)     

> __< p​lowsof:matrix.org >__ briefly checked the prev. 2  MRL meetings logs. Rucknium noted the loose consensus for merging the proposal     

> __< p​lowsof:matrix.org >__ Rucknium: "I think I support this, but it would be much easier to support if it wasn't so expensive. But I know it cannot be done cheaper"     

> __< p​lowsof:matrix.org >__ rbrunner: "I don't see any plausible reason to oppose a CCS" / "it's fine to let donors decide whether they see it worth their donation"     

> __< p​lowsof:matrix.org >__ tobtoht: "Fwiw, I'm in support of getting this audited"     

> __< p​lowsof:matrix.org >__ syntheticbird: "Do it."     

> __< p​lowsof:matrix.org >__ chaser: "it isn't cheap, but given the fact that we will finally get usable multisig in Monero after 10-11 years, it's not expensive either."     

> __< p​lowsof:matrix.org >__ kayabanerve is there a final price now? not much reason to keep it as a draft it seems?     

> __< k​ayabanerve:matrix.org >__ No, I don't have a quote yet :/     

> __< k​ayabanerve:matrix.org >__ I'm hoping to the quote early next week. Then I'll make the CCS, leave it up for comments for however long, and it'll get merged whenever. I brought it up at the last community meeting to get feedback prior to opening the CCS, I was sent to MRL, I got MRL approval, and now I'm here at this upcoming community meeting to get feedback/resolve any other issues I can prior to opening the CCS.     

> __< p​lowsof:matrix.org >__ perfect, thanks     

> __< k​ayabanerve:matrix.org >__ I do get it's a bit silly to have a CCS which isn't a CCS. I don't expect to open the CCS and immediately have it merged. If I had waited to open the CCS, I'd be sent to the MRL in the community meeting on November 9th and only have MRL approval November 20th however. By so pipelining, I hope to save those 25 days.     

> __< k​ayabanerve:matrix.org >__ *If I had waited until I opened the CCS     

> __< p​lowsof:matrix.org >__ makes sense to me 👍️     

> __< m​ainnet_pat:monero.social >__ Updated my CCS proposal draft and made a merge request, hope you guys will have time to check it out https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/513     

> __< plowsof >__ added to the agenda pat, thanks      

> __< plowsof >__ meeting time https://github.com/monero-project/meta/issues/1095      

> __< plowsof >__ Greetings!     

> __< s​yntheticbird:monero.social >__ Hello     

> __< nioCat >__ neow     

> __< m​onerobull:matrix.org >__ hello     

> __< plowsof >__ sharing once again that escapethe3ra has received funding (independently!) for monero.observer until 2026. a recent dev activity report for an overview of the core repos https://monero.observer/monero-dev-activity-report-week-42-2024/      

> __< tobtoht_ >__ hi     

> __< vthor >__ hey :)     

> __< m​onerobull:matrix.org >__ thats very nice     

> __< plowsof >__ https://gui.xmr.pm/files/block.txt is the list selsta maintains , confirmed in this comment https://github.com/monero-project/monero/issues/9526#issuecomment-2429926881     

> __< plowsof >__ boog900 / syntheticbirds contributions have been added there now?      

> __< plowsof >__ hi all     

> __< ofrnxmr >__ Hi jul     

> __< b​oog900:monero.social >__ Hi, yes they have, although a few more have appeared since.     

> __< vthor >__ in the monero docs is written that ban list in normally not applied and need to be enabled (legacy options) when I rememeber right. Is this still like that?     

> __< jberman >__ *waves*     

> __< ofrnxmr >__ it hasnt been reviewed/merged, vthor     

> __< ofrnxmr >__ https://github.com/monero-project/monero-docs/pull/81     

> __< p​lowsof:matrix.org >__ yes, all optional (should be)     

> __< vthor >__ ofrnxmr: but will it be then when it is merged by default be enabled?     

> __< 0​xfffc:monero.social >__ hi everyone     

> __< 0​xfffc:monero.social >__ sorry for the delay     

> __< ofrnxmr >__ 81 pr moves those options from legacy >> relevant sections      

> __< vthor >__ *thumbs up*     

> __< p​lowsof:matrix.org >__ re sharing that the @monero twitter account requests community participation, what/who to tweet/retweet/follow. the matrix channel #monero-twitter:monero.social and #monero-twitter (bridge not set up yet) are available for those who want a say in how the account is handled     

> __< plowsof >__ simplifiedPrivacy wants to share this spanish Monero education series of posts: https://simplifiedprivacy.com/transaccion-monero/index.html     

> __< plowsof >__ https://simplifiedprivacy.com/categories/espanol.html for our spanish users      

> __< s​yntheticbird:monero.social >__ re @monero no one answered my question so it is pretty safe to assume the @monero.bsky.app bluesky handle is actually in the hand of an unknown individual. kinda sad Bluesky is exploding and could have been another network to use.     

> __< plowsof >__ the recent churning research paper by cypherstack: Goodell, Salazar, & Slaughter (2024). “Uniformly Most Powerful Tests for Ad Hoc Transactions in Monero.” has been posted: https://github.com/cypherstack/churn/releases/tag/final      

> __< plowsof >__ [Ruckniums review](https://github.com/cypherstack/churn/issues/2) , SGP noted that after reading the paper / Ruckniums review, he found nothing actionable      

> __< r​ucknium:monero.social >__ My 3 months of server hosting is open for donations: https://rucknium.me/donate/     

> __< p​lowsof:matrix.org >__ @monero.bsky.app is a matrix account?     

> __< s​yntheticbird:monero.social >__ no a bluesky account     

> __< vthor >__ Can I throw here something in what I wanna share, or better after the meeting?     

> __< p​lowsof:matrix.org >__ yes please do     

> __< r​ucknium:monero.social >__ According to kewbit, "Your [Rucknium's] XMR node is the only one I can find that isn’t slow af"  https://libera.monerologs.net/monero-dev/20240824#c416162     

> __< ofrnxmr >__ "i'm getting married!"     

> __< s​yntheticbird:monero.social >__ real     

> __< s​yntheticbird:monero.social >__ absolutely real, rucknium node is the only one that doesn't suck sorry for the term but its real     

> __< ofrnxmr >__ why would rucks be the only one fast? Sounds like user error     

> __< ofrnxmr >__ My android nide is fast     

> __< p​lowsof:matrix.org >__ now that everyone knows, it can be drained of bandwidth as everyone switches, and the cycle continues     

> __< m​onerobull:matrix.org >__ my node is fast too     

> __< vthor >__ I'm still working on it, but I hope to find some input and in some areas help https://moneroroads.com/     

> __< vthor >__ only the header is relevant, rest ist still from the theme I use.     

> __< ofrnxmr >__ what is moneroroads     

> __< plowsof >__ "Join North America's most visible Monero awareness campaign, traveling from Alaska to Cancun. Our 40' rolling billboard combines digital displays with premium advertising space, "     

> __< r​ucknium:monero.social >__ Probably because my node caches about 40GB of the blockchain in RAM, so it serves the data fast to many customers.     

> __< vthor >__ If anybody are interested in setting up a strategy for "deep" adoption onboard I would be happy to share thoughts here: https://github.com/MoneroRoads/MoneroRoads/discussions     

> __< ofrnxmr >__ Plowsof's node caches that much too     

> __< vthor >__ "Probably because my node caches about 40GB of the blockchain in RAM, so it serves the data fast to many customers." <- cool thing!     

> __< ofrnxmr >__ Maybe its due to nobody proxying to rucks node (how many rpc connections do you have, ans how much monthly data does it use ;))     

> __< r​ucknium:monero.social >__ It does the caching by default. The machine just has enough RAM to do that, and lots of requests I guess.     

> __< jberman >__ Also hoping to discuss my CCS, let me know when is a good time to bring it up :)     

> __< s​yntheticbird:monero.social >__ One day cuprate will replace monerod on rucknium node I can guarantee it     

> __< r​ucknium:monero.social >__ I don't check RPC data.     

> __< ofrnxmr >__ Yeah, plowsof has a high-ram node      

> __< vthor >__ "Plowsof's node caches that much too" :o 8)     

> __< p​lowsof:matrix.org >__ i think we can move onto the CCS ideas?     

> __< r​avfx:xmr.mx >__ I allocated 64GB on mine (24 Core Epyc)     

> __< r​avfx:xmr.mx >__ But I reverted it into a normal node     

> __< p​lowsof:matrix.org >__ the malicious proxies require only the best nodes!     

> __< ofrnxmr >__ You can estimate rpc data by just looking at print_net_stats. The diff between up and down is usually (mostly) rpc      

> __< s​yntheticbird:monero.social >__ Rav. You just said on kyun.host that you could cache 256TB of data. Are you elon musk?     

> __< r​avfx:xmr.mx >__ Yeah, that I confirm     

> __< p​lowsof:matrix.org >__ we can move on to the CCS ideas unless there is anything else     

> __< ofrnxmr >__ Yes, lets move on to CCS      

> __< r​avfx:xmr.mx >__ You read in diagonal, you did not get the point     

> __< r​ucknium:monero.social >__ Oh yeah, the server costs are also for moneroresearch.info, which will probably get linked on getmonero.org soon     

> __< s​yntheticbird:monero.social >__ oh you're right sorry     

> __< ofrnxmr >__ Rav is elon, confirmed     

> __< p​lowsof:matrix.org >__ a. [Offline Signing Library for XmrSigner Production](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/495)     

> __< p​lowsof:matrix.org >__ vthors proposal     

> __< tobtoht_ >__ The updated proposal addressed my comments.     

> __< p​lowsof:matrix.org >__ one unique thing about this proposal "Optional strech goals overfunding up to 112" that there has to be consensus on / clearly pointed out , if people want to move forward with the proposal     

> __< p​lowsof:matrix.org >__ thanks for confirming tobtoht, glad to hear     

> __< ofrnxmr >__ What is 112"?      

> __< vthor >__ Thank you plowsof, there I would like to add this link, I tried to get some support from there but seems (and got told) that twitter people probably not gonna show up on meetings or take the effort to visit and register on the ccs page: https://x.com/XmrSigner/status/1847557103419789518 45 likes (the easy way to vote during scrolling....)     

> __< ofrnxmr >__ Likes on twit dont really mean much tbf     

> __< ofrnxmr >__ Being popular get you casual scrollthru likes     

> __< p​lowsof:matrix.org >__ if the proposal is overfunded while on the funding page, then the extra amount, vthor capped it at 112, would go to vthor to work on the same proposal     

> __< vthor >__ 112" * 2.54 = roughly 280 cm     

> __< ofrnxmr >__ Lmao. I read it as 112", not end of quote. Facepalm     

> __< vthor >__ "Likes on twit dont really mean much tbf" for that I said, easy while scrolling... but wanted to mention it...     

> __< p​lowsof:matrix.org >__ so it would go to funding at amount: 196 . then , if it where overfunded (anything over 196 to a max of 112) gets given to vthor 'stretch goals'     

> __< s​yntheticbird:monero.social >__ i vote +1     

> __< ofrnxmr >__ To a max of 112 overfunded? So 298?     

> __< vthor >__ Lmao. I read it as 112", not end of quote. Facepalm :D     

> __< ofrnxmr >__ I think: overfunding should be repurposed within ccs     

> __< p​lowsof:matrix.org >__ one issue would be 'how long is it allowed to stay on the funding page if/when funded' -> fairness... etc theres some details there to be ironed out before people vote on that imo     

> __< p​lowsof:matrix.org >__ but tobtohts comments being addressed is a major plus     

> __< ofrnxmr >__ but not necessarily to the same proposak     

> __< s​yntheticbird:monero.social >__ ok i vote 0     

> __< vthor >__ yes, I could not squize it into the 3 months, I was already trying everywhere to cut time... I hope I'm faster but I want to have time I can chip in unpaid hours if I'm stuck somewhere without getting in to much trouble an stress.     

> __< ofrnxmr >__ and i also think that would obl     

> __< p​lowsof:matrix.org >__ IF the overfunding milestone is contentious , then people can vote on merging without the clause, or, abstain for more clarification or merge as is     

> __< ofrnxmr >__ Only be fair if we paid overfunding back to everyone (like cyjan, who had >100% overfunded)     

> __< vthor >__ Well I can take it out, if this is in the way.     

> __< p​lowsof:matrix.org >__ historical overfunding to be returned to everyone effected ? https://github.com/plowsof/scrape_ccs_fr (including the CCS1 wallet)     

> __< ofrnxmr >__ i'd prefer a clause "overfunding goes to CCS" (instead if generalfund)     

> __< r​avfx:xmr.mx >__ same     

> __< vthor >__ should that then not be changed simply in general?     

> __< m​onerobull:matrix.org >__ i think jeffro deserves his 0.001675431325     

> __< p​lowsof:matrix.org >__ sadly these are topics that could delay vthors proposal     

> __< ofrnxmr >__ vthor: yes. I want it changed in general     

> __< s​yntheticbird:monero.social >__ jeffro is core?     

> __< ofrnxmr >__ No     

> __< m​onerobull:matrix.org >__ i think its best to do case-by case     

> __< vthor >__ So I will change it. No hard feelings on that. Please don't keep me further in the loop :)     

> __< p​lowsof:matrix.org >__ luigi1111 mentioned the one-off instance for sneedlewoods     

> __< ofrnxmr >__ It should never go to GF,  in any case     

> __< s​needlewoods:monero.social >__ to me it seems vthor already put a lot of effort into this, I would give a soft +1     

> __< p​lowsof:matrix.org >__ aka the flood gates are open     

> __< tobtoht_ >__ "i'd prefer a clause" <- Agree. If we set the precedent here, we should change the rules for all future proposals.     

> __< m​onerobull:matrix.org >__ if its "this 1000 XMR proposal was funded by 2 whales sending 1k each" that is different from "this proposal has 1 xmr extra     

> __< ofrnxmr >__ Its not different     

> __< p​lowsof:matrix.org >__ but luigi was on his yacht when this proposal was on the funding page so it was on for longer     

> __< s​yntheticbird:monero.social >__ Whether we accept the overfunding or not I give a +1 for the proposal it is detailed and sound     

> __< p​lowsof:matrix.org >__ or in the case of loveras overfunding - the ccs was broken , and donations didnt show     

> __< m​onerobull:matrix.org >__ it is ofrn     

> __< ofrnxmr >__ no, it really isnt mb     

> __< m​onerobull:matrix.org >__ nobody has infinite resources     

> __< vthor >__  "Agree. If we set the precedent here, we should change the rules for all future proposals." <- so I set it like that?     

> __< ofrnxmr >__ 100xmr or 1xmr, same difference      

> __< m​onerobull:matrix.org >__ and we shouldnt massively overpay if we can use the 1k xmr on something else     

> __< ofrnxmr >__ We shouldnt pay to GF in any case. Should stay within CCS. This is my preference whether 0.01xmr or 100xmr     

> __< m​onerobull:matrix.org >__ agreed     

> __< vthor >__ Can you please on voting consider that I will change overfunding goes to CCS then?     

> __< p​lowsof:matrix.org >__ but vthor wants his proposal merged or not. seems like we cant get a clear vote with the overfunding thing?     

> __< ofrnxmr >__ Overfunding is a sep issue that should change the ccs rule     

> __< p​lowsof:matrix.org >__ remove the clause     

> __< p​lowsof:matrix.org >__ and form a workers union     

> __< p​lowsof:matrix.org >__ let them hadle it     

> __< p​lowsof:matrix.org >__ handle* lol     

> __< ofrnxmr >__ Vthors proposal can likely drop the clause altogether and be voted on w/o it      

> __< vthor >__ plowsof: will do, only can't follow here and change on the web interface at the same time...     

> __< p​lowsof:matrix.org >__ is there loose consensus for merging without the overfunding clause     

> __< tobtoht_ >__ +1 from me without overfunding clause     

> __< ofrnxmr >__ Votes on proposal look good     

> __< ofrnxmr >__ On gitlab*     

> __< m​onerobull:matrix.org >__ +1     

> __< vthor >__ I hope I catched all in the change if not I will correct it later again.     

> __< p​lowsof:matrix.org >__ the tweeter shows positive community engagement, or at least, that it has been revived, oh vthor you forgot to link your overly invasive progrss tracker     

> __< p​lowsof:matrix.org >__ please share the link again, but seems we can merge without the clause     

> __< p​lowsof:matrix.org >__ moving on     

> __< p​lowsof:matrix.org >__ b. [Carrot animated video](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/506)     

> __< vthor >__ :D https://diosdelrayo.github.io/ there will be visible when and how much on which task I'm working. Still fixing real time (needs js)     

> __< p​lowsof:matrix.org >__ this proposal has received no edits/comments from SimplifiedPrivacy.com , only my comment clarifying that they can make videos on ANY topic - i just so happened to suggest the carrot topic     

> __< ofrnxmr >__ we still have a video on the ccs funding page     

> __< p​lowsof:matrix.org >__ it seems video proposal are not gaining as much traction as other proposals https://ccs.getmonero.org/funding-required/ (i can't even make this comment as my own proposal looked like i was being fired before a whale topped it up)     

> __< m​onerobull:matrix.org >__ id rather have vosto than this     

> __< p​lowsof:matrix.org >__ and xenu who has been pushing out content of his own on the youtubes     

> __< s​yntheticbird:monero.social >__ Do you have vosto work examples ? I don't remember having seen any of his videos     

> __< p​lowsof:matrix.org >__ how dare you ask for examples when theyre linked in the proposal. mods?     

> __< ofrnxmr >__ Theyre on getmonero.org faq     

> __< ofrnxmr >__ Didnt you port the website?!!      

> __< m​onerobull:matrix.org >__ https://www.youtube.com/watch?v=RsNOi0lpiyM     

> __< p​lowsof:matrix.org >__ syntheticbird @ https://ccs.getmonero.org/proposals/VOSTOEMISIO-FCMP-Animated-Explainer.html     

> __< p​lowsof:matrix.org >__ i think we can move on?     

> __< s​yntheticbird:monero.social >__ ofrnxmr i did. didn't know it was his work     

> __< p​lowsof:matrix.org >__ lol     

> __< p​lowsof:matrix.org >__ c. [Draft: Audit monero-serai and monero-wallet](https://gist.github.com/kayabaNerve/3723d0a3f2b62ef8ef00c0c4a574fb8e)     

> __< k​ayabanerve:matrix.org >__ Sorry. I thought this started in twenty minutes. Have I missed anything I should've been here for?     

> __< vthor >__ well this video is 2 magnitudes better     

> __< p​lowsof:matrix.org >__ kayabanerves     

> __< k​ayabanerve:matrix.org >__ oh, perfect timing     

> __< s​yntheticbird:monero.social >__ literally perfect timing     

> __< m​onerobull:matrix.org >__ id like to register a permanent +1 on all kayaba proposals good till i change it     

> __< tobtoht_ >__ Voiced support for this during the MRL meeting.      

> __< p​lowsof:matrix.org >__ so the previous 2 MRL meetings , as noted by Rucknium, was a loose consensus for merging (i sprinkled some pings here earlier)     

> __< s​yntheticbird:monero.social >__ in kayabanerve we trust     

> __< jberman >__ I +1'd this as well     

> __< k​ayabanerve:matrix.org >__ That CCS is the draft of the CCS I'd publish once I have the amount finalized (my personal estimate is 750-1000 XMR). The MRL has given their approval *with knowledge of my estimate*.     

> __< p​lowsof:matrix.org >__ linked some of those comments here https://libera.monerologs.net/monero-community/20241026#c451722     

> __< k​ayabanerve:matrix.org >__ They did not solely rule on the content yet also the pricing *assuming the pricing follows my estimate*.     

> __< p​lowsof:matrix.org >__ thanks all for the input and kayabanerve for gathering feedback, i think we can leave it at positive merge sentiment     

> __< p​lowsof:matrix.org >__ hopefully cypherstack have not been handed a blank cheque :)     

> __< vthor >__ I don't know if I have a vote or if it even matters, but I would also +1     

> __< k​ayabanerve:matrix.org >__ Great. In that case, I hope to have the estimate early next week, and I'll post the CCS with it. I presume that'll stay up for whatever comment period luigi feels valuable? But if there's no notable concerns raised, should be smooth smailing :)     

> __< p​lowsof:matrix.org >__ ok thanks, lets move on     

> __< p​lowsof:matrix.org >__ d. [Suggestion for re purposing Outreach funds](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/70#note_26878)     

> __< m​onerobull:matrix.org >__ do we actually have these funds for real or were they stolen and replaced by GF?     

> __< p​lowsof:matrix.org >__ 36.68 "outreach" abandonned funds. geonic suggests they go toward publicity for the upcoming Monerotopia (November 14-17 in Mexico City).     

> __< s​yntheticbird:monero.social >__ What publicity exactly? Who is going to do it?     

> __< s​yntheticbird:monero.social >__ Im all for the intent but it sounds pretty vague obviously     

> __< p​lowsof:matrix.org >__ monerobull: that is coincidentally my recent request, we (the ccs wallet) does not have those funds* they reside in the GF wallet, but, the funds will be returned to the ccs wallet soon.     

> __< m​onerobull:matrix.org >__ imo write them off then     

> __< p​lowsof:matrix.org >__ so yes the ccs wallet has those funds to repurpose*     

> __< m​onerobull:matrix.org >__ we cant just roll the entire theft onto the gf. neros were taken and i dont think we should act like we can just take them from gf     

> __< p​lowsof:matrix.org >__ so that would be a vote for making a merge request to raise funds for monerotopia publicity (being agnostic of the outreach funds from the abandoned proposal?)     

> __< p​lowsof:matrix.org >__ monerobull: thats already happened(happening)     

> __< m​onerobull:matrix.org >__ especially the abandoned funds should just be written off imo     

> __< p​lowsof:matrix.org >__ where do you think the haveno funds are coming from     

> __< m​onerobull:matrix.org >__ hm true     

> __< p​lowsof:matrix.org >__ anyway     

> __< p​lowsof:matrix.org >__ or xmrsigner.. or...     

> __< m​onerobull:matrix.org >__ monerotopia should make a new request then, yeah     

> __< p​lowsof:matrix.org >__ from the linked comment ", I'm asking for preliminary approval to spend up to 36.68 XMR on marketing Monerotopia. A full report with invoices and payments will be provided when requesting payout."     

> __< ofrnxmr >__ I think a new request is the only way to logically claim funds      

> __< s​yntheticbird:monero.social >__ i agree with ofrnxmr.     

> __< s​yntheticbird:monero.social >__ I'm not against repurposing the funds but making a detailed PR would be preferable. I don't see what's urgent to request a preliminary approval     

> __< vthor >__ it is not from monotropia, and it's pretty short on time now. For me it makes sense. But now I need really to ask if I have a vote or if I even should shut up :/     

> __< m​onerobull:matrix.org >__ i think we need at least a blurry outline, for example id say no if the plan is to spend it all on facebook ads     

> __< p​lowsof:matrix.org >__ so create a normal merge request for 36.68 xmr that would be moved to work in progress, rather than funding (if successful)     

> __< ofrnxmr >__ a formal request for an unknown amount      

> __< p​lowsof:matrix.org >__ from the comment: "A public relations agency was recommended and they will be coming back with a proposal by Monday. "     

> __< ofrnxmr >__ And to update the proposal accordingly with info and amounts     

> __< p​lowsof:matrix.org >__ ending with geonic confirming he " would be responsible for running the campaign and Doug is OK with me making the ask."     

> __< ofrnxmr >__ yeah, but asking in a comment isnt a real request     

> __< ofrnxmr >__ need to formalize it and track the discussion on the MR's gitlab     

> __< p​lowsof:matrix.org >__ loose consensus for making a merge request then? seems the comment was made on short notice     

> __< r​ottenwheel:kernal.eu >__ plowsof without Doug or Sunita confirming that such... human being will lead that goal, I would not repurpose or merge a brand-new proposal, regardless of community consensus, invoices, receipts, or prayers.     

> __< p​lowsof:matrix.org >__ the idea of running a publicity fund for monerotopia though? any sentiments to share on that?     

> __< p​lowsof:matrix.org >__ rottenwheel finger on the pulse please     

> __< p​lowsof:matrix.org >__ ah yes, doug needs to confirm it *     

> __< r​ottenwheel:kernal.eu >__ Not against, as long as my condition is met.     

> __< ofrnxmr >__ It depends on the funds are paying for     

> __< p​lowsof:matrix.org >__ geonic has MC'd for them previously though right?     

> __< ofrnxmr >__ Monerokon, not sure about monerotopia     

> __< r​ottenwheel:kernal.eu >__ Supposedly.     

> __< p​lowsof:matrix.org >__ understood rottenwheel thank you     

> __< vthor >__ what means MC'd btw?     

> __< ofrnxmr >__ Mic controller      

> __< nioCat >__ master of ceremonies      

> __< ofrnxmr >__ :D      

> __< p​lowsof:matrix.org >__ lets get ready to rummmblleee!     

> __< p​lowsof:matrix.org >__ iiiiiiiitss tiiiiiime!     

> __< nioCat >__ geo said he did so at topia     

> __< vthor >__ "master of ceremonies" <- serious answer?     

> __< ofrnxmr >__ yes     

> __< p​lowsof:matrix.org >__ so i think we can move on?     

> __< ofrnxmr >__ anyway, i think merge should depend on deliverables      

> __< ofrnxmr >__ The idea of pr for monerotopia is fine, but were paying some outside company for unknown deliverables. This this what needs to be detailed     

> __< nioCat >__ I think he was asking if it was possible and yes depends on what is actually presented     

> __< r​ottenwheel:kernal.eu >__ New merge request with Doug posting a comment confirming he'll be taking care of that endeavor. Done.     

> __< p​lowsof:matrix.org >__ ok moving on     

> __< p​lowsof:matrix.org >__ e. [mainnet_pat CCS proposal for completion of XMR-BCH atomic swaps](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/513)     

> __< nioCat >__ specifically targetting local people     

> __< m​onerobull:matrix.org >__ i saw the monujo webwallet for the first time today     

> __< p​lowsof:matrix.org >__ as this pertains to BCH we must cc Rucknium     

> __< m​onerobull:matrix.org >__ very nice     

> __< m​onerobull:matrix.org >__ has big potential     

> __< m​onerobull:matrix.org >__ especially the walletconnect     

> __< p​lowsof:matrix.org >__ and woodser , who has benefitted (monero-ts) from this work     

> __< m​onerobull:matrix.org >__ could mean easy serai web-ui support for monero     

> __< p​lowsof:matrix.org >__ and ofrnxmr / bsx who have been mentioned in the proposal as previous/ongoing works     

> __< ofrnxmr >__ and bsx who benefitted from his integration      

> __< r​ottenwheel:kernal.eu >__ Against. Clearly swaps are a money milker. ETH one didn't do anything, farcaster, nothing in the wild. BCH? 😂 No. Vapor wave.     

> __< r​ottenwheel:kernal.eu >__ He can open one for the web wallet if he wants.     

> __< ofrnxmr >__ Bch is almost live on basicswap, noob     

> __< ofrnxmr >__ Nothing vapor about it     

> __< r​ottenwheel:kernal.eu >__ The fat ass keeps talking to a wall.     

> __< r​ottenwheel:kernal.eu >__ Cute.     

> __< ofrnxmr >__ s/rottenwheel/rottenwall     

> __< p​lowsof:matrix.org >__ oh just to note, pat has put milestones to unlock the payouts here so there is assurances of tasks being complete     

> __< s​yntheticbird:monero.social >__ thanks god for once again making rotten and ofrn insulting each other during meeting.     

> __< r​ottenwheel:kernal.eu >__ plowsof anyway, the braindead doesn't get there's nobody doing ETH XMR swaps, farcaster wasn't deployed in anything practical, samourai swaps were shut down. COMIT is the only one somewhat with something that works and there is already a hefty CCS for hundreds of XMR for unstoppable.     

> __< p​lowsof:matrix.org >__ a little obscured at the moment due to front matter errors but theyre listed here https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/513/diffs     

> __< ofrnxmr >__ where did i insult anyone..     

> __< r​ottenwheel:kernal.eu >__ I see zero need for a XMR BCH atomic swap proposal.     

> __< s​yntheticbird:monero.social >__ The more the better     

> __< s​yntheticbird:monero.social >__ imagien if litecoin is said to be a privacy coin (same way Dash was one)     

> __< ofrnxmr >__ the protocol was created by a monero bounty      

> __< m​onerobull:matrix.org >__ imo working on improving the webwallet is more useful than a bch atomic swap     

> __< r​ottenwheel:kernal.eu >__ Besides the fact nobody cares about BCH but ruck and three other bcashers. 😅     

> __< p​lowsof:matrix.org >__ you said "bcash" and revealed the bias, sorry !     

> __< s​yntheticbird:monero.social >__ I hope the web wallet is pretty     

> __< p​lowsof:matrix.org >__ bitcoin maxi alert     

> __< ofrnxmr >__ monujo.cash      

> __< r​ottenwheel:kernal.eu >__ I wasn't trying to hide it lol.     

> __< ofrnxmr >__ Syn ^     

> __< r​ucknium:monero.social >__ On the additional value of this proposal: AFAIK, Basicswap requires installing programs on a desktop and running the nodes of the chains that the user wants to swap with. mainnet_pat's proposal is a web UI that will be easier for users. mainnet_pat also has made a lot of contributions to `monero-ts` and later he could be one of the knowledgeable users of it to help more projects.     

> __< s​yntheticbird:monero.social >__ thx ofrnxmr     

> __< ofrnxmr >__ Basicswap is a desktop program, yes, and currently relies on local nodes to remain trustless     

> __< s​yntheticbird:monero.social >__ what if I want to trust ruck node?     

> __< s​yntheticbird:monero.social >__ censorship     

> __< ofrnxmr >__ Local nodes arent a hard requirement if we were to offload checks elsewhere     

> __< ofrnxmr >__ You can use monero remote nodes, but other chains use core wallets     

> __< r​ucknium:monero.social >__ The final milestone should be a production-ready user-facing app, to make sure that this won't just be a proof-of-concept.; The BCH community funded a lot of the work so far. pat underestimated the amount f work involved, so he is asking CCS to fund the final leg of the development.     

> __< ofrnxmr >__ (not electrum/spv)     

> __< s​yntheticbird:monero.social >__ ofrnxmr 👍     

> __< p​lowsof:matrix.org >__ thanks for the feedback Rucknium     

> __< r​ucknium:monero.social >__ ETH<>XMR atomic swaps never got a UI. It's a CLI only for now.     

> __< m​onerobull:matrix.org >__ monujo looks like a really good base to use for serai frontend integration     

> __< ofrnxmr >__ Note that pat has completed probably 80-90% of the integration of bch<>xmr swap protocol into bsx and should be live soon     

> __< k​ayabanerve:matrix.org >__ CLIs are UIs Rucknium     

> __< k​ayabanerve:matrix.org >__ skill issue on your end if you disagree     

> __< k​ayabanerve:matrix.org >__ /s :P     

> __< m​onerobull:matrix.org >__ Thorswap has walletconnect but they would probably never add a full monero wallet with scanning     

> __< ofrnxmr >__ :D     

> __< ofrnxmr >__ What im saying, is that i'm pleased with his work     

> __< r​ucknium:monero.social >__ It may be reasonable to see what the liquidity on BasicSwap XMR<>BCH looks like before this is funded. Note: This web app's order book and the BasicSwap order book is expected to be separate     

> __< l​ordx3nu:matrix.org >__ Me and vosto can do something re: carrot     

> __< p​lowsof:matrix.org >__ ofrnxmr any estimates on bch-xmr swaps going live on BSX? or too early to tell?     

> __< p​lowsof:matrix.org >__ in the interest of being inclusive i will mention bcash for every bch i mention     

> __< ofrnxmr >__ i think this week it should be finished and rdy for review, then live on dev branch for testing      

> __< p​lowsof:matrix.org >__ so Ruckniums suggestion of waiting to see the liquidity there could be realistic hm     

> __< m​ainnet_pat:monero.social >__ I wanted to abstain from commenting. but this is a bit controversial, judging on one product upon another product with different characteristics is like comparing apples and oranges. also tying the approval of one upon the performance of other (a competitor one?) is questionable.     

> __< m​ainnet_pat:monero.social >__ wrt orderbooks it is obvious that no one will be targeting the bsx orderbooks based on smgs since bsx itself wants to phase away from using smsg.     

> __< m​ainnet_pat:monero.social >__ also wrt orderbooks:     

> __< m​ainnet_pat:monero.social >__ Both apps have non-committing orderbooks, so funds are not locked in any way and orderbook orders are just intentions to trade, not commitments to trade. So a user with X amount of XMR may participate with the same X amounts in both orderbooks. This is very technical and I do not think it should be included in the proposal     

> __< m​ainnet_pat:monero.social >__ I wanted to abstain from commenting. but this is a bit controversial: judging on one product upon another product with different characteristics is like comparing apples and oranges. also tying the approval of one upon the performance of other (a competitor one?) is questionable.     

> __< m​ainnet_pat:monero.social >__ wrt orderbooks it is obvious that no one will be targeting the bsx orderbooks based on smgs since bsx itself wants to phase away from using smsg.     

> __< m​ainnet_pat:monero.social >__ also wrt orderbooks:     

> __< m​ainnet_pat:monero.social >__ Both apps have non-committing orderbooks, so funds are not locked in any way and orderbook orders are just intentions to trade, not commitments to trade. So a user with X amount of XMR may participate with the same X amounts in both orderbooks. This is very technical and I do not think it should be included in the proposal     

> __< ofrnxmr >__ Please no edits 😅     

> __< r​ucknium:monero.social >__ I still think Monujo is not a good name for a wallet and Diego Salazar  should help rebrand it. But it's ok if pat really likes the Monujo name.     

> __< ofrnxmr >__ Spamming irc     

> __< m​ainnet_pat:monero.social >__ substituted comma by colon ^     

> __< p​lowsof:matrix.org >__ "tying the approval of one upon the performance of other (a competitor one?) is questionable." good point reg waiting to see how the swaps go with bsx hm     

> __< ofrnxmr >__ I think rucks comments is more about "will anyone provide liquidity for the pair?"     

> __< m​ainnet_pat:monero.social >__ and liquidity comes when marketmakers get comfy bots with plug and play     

> __< p​lowsof:matrix.org >__ the xmr btc liquidity for https://unstoppableswap.net/ has a graph. seems waiting several months to see how it goes would be unrealistic then     

> __< d​iego:cypherstack.com >__ Hi its me     

> __< p​lowsof:matrix.org >__ although a functional . easy to use GUI would give xmr/bch swaps a head start but still     

> __< d​iego:cypherstack.com >__ I made some suggestions before on the name.     

> __< m​ainnet_pat:monero.social >__ I have seen you suggestions and I am thankful for them. Monujo and monerujo sounds different for me, so I decided to postpone the decision     

> __< p​lowsof:matrix.org >__ sadly "unnamed wallet" is taken by Czarek Nakamoto     

> __< m​ainnet_pat:monero.social >__ one of my goals is to transfer ownership of my privacy developments to some organization where I would be coder, not an owner to not be hunted by 3letter agencies like samourai devs     

> __< m​ainnet_pat:monero.social >__ one of my goals is to transfer ownership of my privacy developments to some organization where I would be coder, not an owner, to not be hunted by 3letter agencies like samourai devs     

> __< ofrnxmr >__ Please try to avoid edits     

> __< m​ainnet_pat:monero.social >__ I am sorry, it is a habit, I will take extra care     

> __< p​lowsof:matrix.org >__ i think we can put an open ending to the meeting while talk continues, as we're 23 mins over     

> __< jberman >__ Ok, I'm going to share my thing     

> __< p​lowsof:matrix.org >__ AOB / all ears     

> __< jberman >__ CCS update: after discussing in the NWLB channel about a month ago, I started working on full wallet sync for fcmp++ (while waiting on further fcmp++ prove/verify work from @kayabanerve, and since dangerousfreedom decided to take a hiatus working on monero)     

> __< jberman >__ Full wallet sync is a necessary component we'd eventually need for the fcmp++ integration     

> __< jberman >__ Since I've spent quite some time on full wallet sync, there is a chance I won't be able to deliver all of fcmp++ tx construction, verification, and consensus changes within the allotted hours of my CCS     

> __< jberman >__ My CCS did not include implementing wallet sync as a deliverable. It did include fcmp++ tx construction, verification, and consensus changes     

> __< jberman >__ I bring this up hoping to give a far-in-advance heads up of this possibility and get feedback on it. I brought it up in the last NWLB meting to get feedback there too, and generally got a thumbs up     

> __< jberman >__ My CCS: https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/491     

> __< jberman >__ Initial discussion to switch tasks in NWLB channel: https://github.com/monero-project/meta/issues/1080     

> __< jberman >__ Most recent discussion mentioning I may end up switching deliverables for my current CCS: https://github.com/monero-project/meta/issues/1094     

> __< ofrnxmr >__ dangerous freedoms funds are up for grabs      

> __< tobtoht_ >__ Strong support for allowing j-berman to credit hours worked on wallet sync to the current CCS proposal, and to open a new CCS to continue fork on the fcmp++ integration.     

> __< ofrnxmr >__ Wouldnt need to switch tasks, can originally claim DF's funds (?)     

> __< ofrnxmr >__ Probably* claim     

> __< tobtoht_ >__ s/fork/work/     

> __< ofrnxmr >__ wouldn't it make more sense to open an impromptu 2nd ccs to chain DF's funds for the work? and to just delay the FCMP++ milestone     

> __< ofrnxmr >__ s/chain/claim     

> __< jberman >__ first, dangerous freedom's current CCS doesn't mention this task as part of the scope of their CCS. second, DF's rate was lower than mine     

> __< p​lowsof:matrix.org >__ DF is on vacation, leave him alone!     

> __< ofrnxmr >__ DF asked us to use his funds     

> __< ofrnxmr >__ To put them to good use*     

> __< jberman >__ this is a FCMP++ milestone fwiw     

> __< nioCat >__ I followed the discussion in the NWLB meeting     

> __< nioCat >__ +1     

> __< p​lowsof:matrix.org >__ jberman's ccs are batches of hours worked. seems sane to re-up on those hours with a new merge request     

> __< ofrnxmr >__ So leave DF funds alone?      

> __< ofrnxmr >__ Save for another rainy day?     

> __< p​lowsof:matrix.org >__ i need to find the exact interation i had with him     

> __< nioCat >__ save for when he comes back      

> __< p​lowsof:matrix.org >__ if he said "use them" or "use them if ill be cancelled" . i mentioned that some proposals are open for x years and leaving for x months is fine     

> __< ofrnxmr >__ he asked us to put them to good use      

> __< ofrnxmr >__ In NWLB     

> __< m​onerobull:monero.social >__ This is bcs he got external funding right     

> __< nioCat >__ yes but others said to save them for when he comes back     

> __< nioCat >__ incentive?     

> __< nioCat >__ if he doesn't come back then something can be done with the xmr     

> __< ofrnxmr >__ How long do we wait 😆     

> __< nioCat >__ 1 year     

> __< plowsof >__ i said "note for dangerousfreedom: there are other ccs proposals who have been open for 2~3 years , so 1 year isn't enough to start distributing funds ^^ " , need to see his response     

> __< ofrnxmr >__ So, I engaged in some personal projects and won't be able to dedicate much time to Monero for one year at least. Therefore, I have to give up my CCS and apologize to everyone who donated for it. I'm sure plowsof  and ofrnxmr  will find good projects to redirect the funds. I dont feel really bad because the project changed quite a bit since I     

> __< ofrnxmr >__ proposed and I'm sure jeffro256 , jberman , SNeedlewoods  , kayabanerve , tobtoht , rbrunner7 and the others here will deliver a great wallet for the FCMP update. With that said, it means that I won't be active in the group meetings anymore but will be improving my skills and will try to contribute whenever I can. In the end privacy wins, otherwise     

> __< ofrnxmr >__ it is not the end yet :)     

> __< ofrnxmr >__ Nioc, he said he'd be gone for 1 yr at least, and said to find good projects to redirect the funds      

> __< m​onerobull:monero.social >__ Wow this comes out of nowhere     

> __< nioCat >__ yes and there was discussion after he stated that     

> __< ofrnxmr >__ This was on sept 9     

> __< ofrnxmr >__ Nioc, did he respond?      

> __< nioCat >__ I am not sure     

> __< ofrnxmr >__ "Before I go, I want to emphasize that it was really a pleasure meeting each of you. I have worked in some places and traveled quite a bit and can say that it is really hard to see brilliant and dedicated people like you guys. Keep strong on your side and I will keep improving on my side. See you someday ;)" this doesnt sound like "i'll be back"     

> __< nioCat >__ the only way for him to see us is for him to come back   lol     

> __< nioCat >__ anyway     

> __< ofrnxmr >__ anyway, he asked for the ccs to be closed and repurposed and looks like he never entertained any attempt at otherwise      

> __< nioCat >__ he was being considerate and yes nothing is certain     

> __< ofrnxmr >__ I'm not assuming he wont come back, but i'm assuming he's not planning on finishing the work     

> __< ofrnxmr >__ And wants the funds to be allocated to the someone else who will do the work     

> __< nioCat >__ who else?  :)     

> __< jberman >__ I would feel a little uncomfortable taking his funds personally. I also felt there was a window of "I'll be back" in the discussion, especially from "will be improving my skills and will try to contribute whenever I can" and "it is not the end yet"     

> __< jberman >__ I wouldn't protest the decision, but figured worth sharing my thoughts regardless     

> __< nioCat >__ let's take some time making this decision  :D     

> __< jberman >__ Does feel a little tangential to my main request though, which is basically what tobtoht_ said: if it's acceptable to credit hours working on wallet sync towards my current ccs (at my rate), and upon completing the hours in this ccs, continue on fcmp++ integration (which would include working on the deliverables of my current CCS)     

> __< jberman >__ Would continue in a follow-up ccs     

> __< m​onerobull:monero.social >__ Ok next topic?     

> __< nioCat >__ all work that needed to be done, seems like an easy +1     

> __< m​onerobull:monero.social >__ Tomorrow in the website meeting we will vote on the new design for good     

> __< m​onerobull:matrix.org >__ I guess we are done here for good, see you tomorrow     

> __< nioCat >__ @seen plowsof      

> __< r​ottenwheel:kernal.eu >__ He ended the meeting half an hour ago... 🙄     

> __< plowsof >__ i needed a food break, sorry, EOF, who wants a cup of tea      


Automated by [this](https://github.com/plowsof/post-libera-meeting-logs)
